api/config/lighthouse.php

```diff
@@ -29,84 +30,113 @@
          * make sure to return spec-compliant responses in case an error is thrown.
          */
         'middleware' => [
-            \Nuwave\Lighthouse\Support\Http\Middleware\AcceptJson::class,
+            // Ensures the request is not vulnerable to cross-site request forgery.
+            // Nuwave\Lighthouse\Http\Middleware\EnsureXHR::class,
+
+            // Always set the `Accept: application/json` header.
+            Nuwave\Lighthouse\Http\Middleware\AcceptJson::class,
 
             // Logs in a user if they are authenticated. In contrast to Laravel's 'auth'
             // middleware, this delegates auth and permission checks to the field level.
-            \Nuwave\Lighthouse\Support\Http\Middleware\AttemptAuthentication::class,
+            Nuwave\Lighthouse\Http\Middleware\AttemptAuthentication::class,
 
             // Logs every incoming GraphQL query.
-            App\Http\Middleware\AuditQueryMiddleware::class,
+            Nuwave\Lighthouse\Http\Middleware\LogGraphQLQueries::class,
 
```

Comment:
Did you mean to remove our AuditQueryMiddleware middleware?

Reply:
Yeah, as far as I could tell `LogGraphQLQueries::class` (new to v6) was nearly identical so I preferred that over the custom `AuditQuryMiddleware`. If I was incorrect in this assumption, let me know and I will revert that change 👍

Reply:
The main difference is that AuditQueryMiddleware only logs queries from admins.  If we start logging queries from everyone our log usage will shoot up, I think.

Reply:
That makes total sense, thanks! Reverted back and added a comment to distinguish the two in [add query audit middleware back](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7434/commits/7ef5e9366ecb636cebdac7170c03b8f25dbf0f95)